sklearn/tests/test_pipeline.py

```diff
@@ -125,6 +131,17 @@ def score(self, X, y=None, sample_weight=None):
         return np.sum(X)
 
 
+class DummyTransf(Transf):
+    """Transformer which store the column means"""
+
+    def fit(self, X, y):
+        self.means_ = np.mean(X, axis=0)
+        # Store a timestamp such that we know
+        # that we have a cache object
```

Comment:
Nitpick: store timestamp to figure out whether the result of 'fit' has been cached or not